payroll-processor.code-workspace

```diff
@@ -1,23 +1,55 @@
 {
-	"folders": [
-		{
-			"path": "payroll-processor-api",
-			"name": "api"
-		},
-		{
-			"path": "payroll-processor-client",
-			"name": "client"
-		},
-		{
-			"path": "payroll-processor-functions",
-			"name": "funtions"
-		},
-		{
-			"path": ".",
-			"name": "root"
-		}
-	],
-	"settings": {
+  "folders": [
+    {
+      "path": "payroll-processor-api",
+      "name": "api"
+    },
+    {
+      "path": "payroll-processor-client",
+      "name": "client"
+    },
+    {
+      "path": "payroll-processor-functions",
+      "name": "funtions"
+    },
+    {
+      "path": ".",
+      "name": "root"
+    }
+  ],
+  "settings": {
+    "files.exclude": {
+      "**/.git": true,
+      "**/.svn": true,
+      "**/.hg": true,
+      "**/CVS": true,
+      "**/.DS_Store": true,
+      "node_modules": true
+    },
 
-	}
-}
\ No newline at end of file
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.codeActionsOnSave": {
+      "source.fixAll": true,
+      "source.organizeImports": true
+    },
+
+    "[csharp]": {
+      "editor.defaultFormatter": "ms-dotnettools.csharp"
+    },
+
+    "debug.internalConsoleOptions": "neverOpen"
+  },
+  "extensions": {
+    "recommendations": [
```

Comment:
These are all the extensions that should be installed to make the workspace function correctly